docs/src/modules/components/AppFrame.js

```diff
@@ -206,6 +206,7 @@ class AppFrame extends React.Component {
                       color="inherit"
                       aria-label="Edit docs colors"
                       component={Link}
+                      naked
```

Comment:
Remove the dependency on the CSS injection order.

![capture d ecran 2019-01-12 a 13 14 20](https://user-images.githubusercontent.com/3165635/51073069-1076a200-166c-11e9-8618-8a898957a5ea.png)